vyper/compiler/phases.py

```diff
@@ -184,6 +193,19 @@ def generate_ast(source_code: str, source_id: int, contract_name: str) -> vy_ast
     return vy_ast.parse_to_ast(source_code, source_id, contract_name)
 
 
+def generate_unfolded_ast(
```

Comment:
could you add a comment describing that the purpose of this phase is to generate an AST for tooling use?

now that i think about it, i wonder if it breaks other downstream tools. is it possible to check, e.g. whether brownie still works with the unfolded AST?